source/crud/crud.rst

```diff
@@ -168,6 +170,10 @@ Read
     /**
      * Finds the distinct values for a specified field across a single collection.
      *
+     * Note: Unlike other read methods (e.g. aggregate, find), this method will not return
+     * a cursor. The Iterable<any> return type corresponds to the "values" array in the
+     * distinct command's result document.
```

Comment:
@jyemin: Here, would you prefer I add something to the effect of:

```
Note: As of MongoDB 4.0, the Iterable<any> return type corresponds to the "values"
array in the distinct command's result document. This may change in the future if the
result becomes backed by a cursor (similar to aggregate and find).
```